### PR TITLE
Update openvpn-gui-res-fr.rc

### PR DESCRIPTION
**Pull request by bubuleur**
Hello
attached the update of the French translation of OpenVPN how to see if my translation is good? Thanks

### CODE REVIEW

**Comment by lstipakov** [comment]
Merci beaucoup! I'll ask a French-speaking colleague to check it. Meanwhile there is at least one issue:

` IDS_NFO_STATE_DISCONNECTING Déconnexion"`

The opening quote mark is missing.

File: res/openvpn-gui-res-fr.rc [hunk 1]
```diff
@@ -560,43 +560,43 @@ en tant qu'Administrator pour mettre à jour la base de registre."
...
     IDS_NFO_AUTH_PASS_RETRY "Nom d'utilisateur ou mot de passe incorrect. Réessayer..."
     IDS_NFO_KEY_PASS_RETRY  "Mauvais mot de passe. Réessayer..."
-    IDS_ERR_INVALID_PASSWORD_INPUT "Invalid character in password"
-    IDS_ERR_INVALID_USERNAME_INPUT "Invalid character in username"
-    IDS_NFO_AUTO_CONNECT    "Connecting automatically in %u seconds…"
-    IDS_NFO_CLICK_HERE_TO_START "OpenVPN GUI is already running. Right click on the tray icon to start."
-    IDS_NFO_BYTECOUNT "Bytes in: %ls  out: %ls"
+    IDS_ERR_INVALID_PASSWORD_INPUT "Caractère invalide dans le mot de passe"
+    IDS_ERR_INVALID_USERNAME_INPUT "Caractère invalide dans le nom d'utilisateur"
+    IDS_NFO_AUTO_CONNECT    "Connexion automatique dans %u secondes…"
```
Comment (lstipakov) [inline res/openvpn-gui-res-fr.rc:565]: I would probably think that 'connecter automatiquement ...' is better phrase to use

File: res/openvpn-gui-res-fr.rc [hunk 2]
```diff
@@ -560,43 +560,43 @@ en tant qu'Administrator pour mettre à jour la base de registre."
...
-    IDS_NFO_OVPN_STATE_RECONNECTING "Reconnecting"
-    IDS_NFO_OVPN_STATE_EXITING      "Exiting"
-    IDS_NFO_OVPN_STATE_WAIT         "Waiting"
-    IDS_NFO_OVPN_STATE_AUTH         "Authenticating"
-    IDS_NFO_OVPN_STATE_GET_CONFIG   "Pulling configuration from server"
-    IDS_NFO_OVPN_STATE_RESOLVE      "Resolving remote host"
-    IDS_NFO_OVPN_STATE_TCP_CONNECT  "Establishing TCP connection"
-    IDS_NFO_OVPN_STATE_AUTH_PENDING "Authentication pending"
+    IDS_NFO_OVPN_STATE_INITIAL      "Initialisation"
+    IDS_NFO_OVPN_STATE_CONNECTING   "Connexion"
```
Comment (lstipakov) [inline res/openvpn-gui-res-fr.rc:588]: same as above - Connecter

File: res/openvpn-gui-res-fr.rc [hunk 3]
```diff
@@ -560,43 +560,43 @@ en tant qu'Administrator pour mettre à jour la base de registre."
...
+    IDS_NFO_OVPN_STATE_ASSIGN_IP    "Attribution d'une adresse IP"
+    IDS_NFO_OVPN_STATE_ADD_ROUTES   "Ajouter routes"
+    IDS_NFO_OVPN_STATE_CONNECTED    "Connecté"
+    IDS_NFO_OVPN_STATE_RECONNECTING "Reconnexion"
```
Comment (lstipakov) [inline res/openvpn-gui-res-fr.rc:592]: Reconnecter

File: res/openvpn-gui-res-fr.rc [hunk 4]
```diff
@@ -560,43 +560,43 @@ en tant qu'Administrator pour mettre à jour la base de registre."
...
+    IDS_NFO_OVPN_STATE_EXITING      "Sortie"
```
Comment (lstipakov) [inline res/openvpn-gui-res-fr.rc:593]: Sortir

File: res/openvpn-gui-res-fr.rc [hunk 5]
```diff
@@ -560,43 +560,43 @@ en tant qu'Administrator pour mettre à jour la base de registre."
...
+    IDS_NFO_OVPN_STATE_WAIT         "En attente"
+    IDS_NFO_OVPN_STATE_AUTH         "Authentification"
+    IDS_NFO_OVPN_STATE_GET_CONFIG   "Extraire la configuration du serveur"
+    IDS_NFO_OVPN_STATE_RESOLVE      "Résolution de l'hôte distant"
```
Comment (lstipakov) [inline res/openvpn-gui-res-fr.rc:597]: Résolvir de...

File: res/openvpn-gui-res-fr.rc [hunk 6]
```diff
@@ -560,43 +560,43 @@ en tant qu'Administrator pour mettre à jour la base de registre."
...
+    IDS_NFO_OVPN_STATE_TCP_CONNECT  "Établissement de la connexion TCP"
```
Comment (lstipakov) [inline res/openvpn-gui-res-fr.rc:598]: établir de...

**Comment by bubuleur** [comment]
> Merci beaucoup! I'll ask a French-speaking colleague to check it. Meanwhile there is at least one issue:
> 
> ` IDS_NFO_STATE_DISCONNECTING Déconnexion"`
> 
> The opening quote mark is missing.

Merci à toi pour tes remarques

File: res/openvpn-gui-res-fr.rc (hunk 6 above)
Comment (bubuleur) [reply res/openvpn-gui-res-fr.rc]: Excuse me I did not understand too much

File: res/openvpn-gui-res-fr.rc (hunk 5 above)
Comment (bubuleur) [reply res/openvpn-gui-res-fr.rc:597]: Excuse me I did not understand too much

**Comment by bubuleur** [comment]
Do you know how I can see directly on the software if my translation is good?

**Comment by selvanair** [comment]
You can download the GHA build executable that includes your PR from this link.
https://github.com/OpenVPN/openvpn-gui/actions/runs/5715781857#artifacts
If you are on 64 bit Windows use the one named openvpn-gui_x64 -- otherwise choose an appropriate one from the list.

**Review by lstipakov — APPROVED** [review]
I see that most of suggestions are addressed and some translations have changed a bit. Looks good to me.

**Comment by lstipakov** [comment]
@selvanair @flichtenheld shall we squash and merge?

**Comment by selvanair** [comment]
> I changed some words because on the software you don't see the full words

We can change the text box sizes to accommodate longer phrases. I will take a quick look.

**Comment by selvanair** [comment]
All look good except the help window where the last line is truncated (the one that pops up when `openvpn-gui.exe --help` is run from the command line). But, there are a number of outdated options such as `allow_edit` and `allow_service` in the help text removing which will fix this. I'll follow with a patch to remove obsolete options in the help text in all languages.

+1 to squash and merge this.

**Pull request merged**